scripts/linters/test_files/invalid_sorted_dependencies.ts

```diff
@@ -23,7 +23,7 @@ angular.module('oppia').controller('SuggestionModalForCreatorViewController', [
   'stateName', 'suggestionIsHandled', 'suggestionStatus',
   'suggestionType', 'IMPORT_STATEMENT',
   function(
-      $scope, SuggestionModalService, $uibModalInstance,
+      $uibModalInstance, $scope, SuggestionModalService,
```

Comment:
Any specific reason for these changes?

Reply:
Yes, with those changes linter was raising two errors, as the line break is not at the right place, so to fix that error, I had to make this change.